ci/test/04_install.sh

```diff
@@ -66,8 +66,7 @@ fi
 if [[ $DOCKER_NAME_TAG == centos* ]]; then
   ${CI_RETRY_EXE} DOCKER_EXEC yum -y install epel-release
   ${CI_RETRY_EXE} DOCKER_EXEC yum -y install $DOCKER_PACKAGES $PACKAGES
-elif [ "$TRAVIS_OS_NAME" != "osx" ]; then
-  ${CI_RETRY_EXE} DOCKER_EXEC apt-get update
+elif [ $USE_TRAVIS_APT_ADDON != "yes" ]; then
```

Comment:
```suggestion
elif [ $CI_USE_APT_INSTALL != "no" ]; then
```

Let's try to avoid making this travis specific

Reply:
Done.